AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/BundleHelper.java

```diff
@@ -172,6 +173,10 @@ public BundleHelper(AndroidAnnotationsEnvironment environment, TypeMirror elemen
 			if (isTypeParcelable(type)) {
 				methodNameToSave = "put" + "Parcelable";
 				methodNameToRestore = "get" + "Parcelable";
+			} else if (annotationHelper.isAnnotatedWith(type, "org.parceler.Parcel")) {
```

Comment:
If you can add it, it would be great. However we should also check that we have `Parceler` on the classpath (when the user adds a `List` of primitives for examples, we cannot be sure because we are not checking any annotation).